dev/com.ibm.ws.microprofile.config.1.3_fat/publish/servers/HotAddMPConfig/server.env

```diff
@@ -0,0 +1,7 @@
+MPCONFIG_FATTEST_ENV_VARIABLE=mpconfigtestValue
+__MPCONFIG_FATTEST_ENV_VARIABLE=__mpconfigtestValue
+MPCONFIG_FATTEST_ENV_VARIABLE__=mpconfigtestValue__
+MPCONFIG_FATTEST__ENV_VARIABLE=mpconfig__testValue
+MPCONFIGFATTESTENVVARIABLE=mpconfigtestValue
+mpconfig_lowcase_fattest_env_variable=mpconfiglowcasetestValue
+mpconfig_MIXEDcase_fattest_env_variable=mpconfigmixedcasetestValue
```

Comment:
Don't think this file is related to the test?

Reply:
gone